### ISSUE TITLE
panic in try from slice

### ISSUE THREAD

**Issue by robjtede** [BurntSushi/byteorder#173]
Looks like something panics that didn't before 1.4.0.

```
thread 'actix-rt:worker:0' panicked at 'called `Result::unwrap()` on an `Err` value: TryFromSliceError(())', /home/jens/.cargo/registry/src/github.com-1ecc6299db9ec823/byteorder-1.4.0/src/lib.rs:2185:43
```

Related to:
https://github.com/actix/actix-web/issues/1890

for those finding this, workaround in Cargo.toml, add:
```toml
byteorder = "~1.3"
```

**Comment by robjtede** [comment]
Most basic test case that works in 1.3 and fails in 1.4:

```rust
byteorder::BigEndian::read_u16(&[1, 2, 3]);
```

**Comment by pimeys** [comment]
Yeah. We also have now our build pipelines red because some of the deps is going with `version = "1"`, got upgraded to 1.4.0 and fails hard now...

**Comment by BurntSushi** [comment]
This has been fixed in `byteorder 1.4.1`.

If you want reproducible builds, then you need to use `Cargo.lock`.

**Comment by pimeys** [comment]
We do in the binaries, but this was one library that didn't have one. And I just happened to build when this upgrade arrived!
